src/wkt/src/any.rs

```diff
@@ -136,10 +149,24 @@ impl crate::message::Message for Any {
     fn typename() -> &'static str {
         "type.googleapis.com/google.protobuf.Any"
     }
-    fn to_map(&self) -> Result<crate::message::Map, AnyError> {
-        crate::message::to_json_other(self)
+
+    // Override the default serializer to use custom serialization
+    fn serializer() -> Box<dyn crate::message::MessageSerializer<Self>>
+    where
+        Self: Sized,
+    {
+        Box::new(AnySerializer)
     }
-    fn from_map(map: &crate::message::Map) -> Result<Self, AnyError> {
+}
+
+struct AnySerializer;
```

Comment:
... and then we can use `crate::message::ValueSerializer<Any>` here, instead of repeating ourselves.

Reply:
Done.